### PR TITLE
Nuking OAuth/OIDC Standards pages

### PR DESCRIPTION
**Pull request by jakubvul**
## Description:

- Moving content from Standards pages to new locations
- Deleting old pages
- Creating redirects from old pages to OIDC API Reference

### Resolves:

* [OKTA-171268](https://oktainc.atlassian.net/browse/OKTA-171268)

### CODE REVIEW

File: _source/_authentication-guide/auth-overview/index.md [hunk 1]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+At its core, an authorization server is simply an engine for minting OpenID Connect or OAuth 2.0 tokens. You can't mix tokens between different authorization servers. By design, authorization servers don't have trust relationships with each other.
+
+Okta provides two types of authorization servers:
+
+**Okta Authorization Server**
+
+Use the Okta Authorization Server to perform SSO with Okta or sign in users for apps displayed on the Okta home page.
+Okta hosts and manages the Okta Authorization Server. It can't be configured, though you can add a [groups claim](/docs/api/resources/oidc#scope-dependent-claims-not-always-returned) and [app-user profile attributes](/docs/api/resources/apps#application-user-properties) to a client. The access token minted by the Okta Authorization Server is consumed by Okta APIs. The access token audience is always Okta specific, so the token can't be validated by your applications.
+
+> Note: The Okta Authorization Server doesn't require that the API Access Management feature be enabled.
```
Comment (jakubvul) [inline _source/_authentication-guide/auth-overview/index.md]: @nbarbettini any thoughts on how to handle this? It seems like this sentence needs to at least be updated. We don't really talk about "API Access Management" in the Auth Guide.

File: _source/_authentication-guide/auth-overview/index.md [hunk 2]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+
+## Authorization Servers
+
+At its core, an authorization server is simply an engine for minting OpenID Connect or OAuth 2.0 tokens. You can't mix tokens between different authorization servers. By design, authorization servers don't have trust relationships with each other.
+
+Okta provides two types of authorization servers:
+
+**Okta Authorization Server**
+
+Use the Okta Authorization Server to perform SSO with Okta or sign in users for apps displayed on the Okta home page.
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Suggestion (to replace 284-285):

Use the Okta Authorization Server to perform Single Sign-On with Okta, or get an access token for Okta. The Okta Authorization Server can't be customized. Access tokens issued by the Okta Authorization Server can only be consumed and validated by Okta. The token audience is Okta-specific, so the token can't be used or validated by your own applications.

File: _source/_authentication-guide/auth-overview/index.md (hunk 1 above)
Comment (nbarbettini) [reply _source/_authentication-guide/auth-overview/index.md]: My preference is to remove. This isn't the right place to talk about packaging. I'll get the PMM folks to double check my reasoning. 🙂

File: _source/_authentication-guide/auth-overview/index.md [hunk 3]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+Okta hosts and manages the Okta Authorization Server. It can't be configured, though you can add a [groups claim](/docs/api/resources/oidc#scope-dependent-claims-not-always-returned) and [app-user profile attributes](/docs/api/resources/apps#application-user-properties) to a client. The access token minted by the Okta Authorization Server is consumed by Okta APIs. The access token audience is always Okta specific, so the token can't be validated by your applications.
+
+> Note: The Okta Authorization Server doesn't require that the API Access Management feature be enabled.
+
+**Custom Authorization Server**
+
+Use a Custom Authorization Server to secure your APIs. Custom Authorization Servers are hosted on Okta, and created and configured by an org administrator. An access token minted by a Custom Authorization Server is consumed by your APIs. Custom scopes can be configured to support authorization for your APIs.
+
+> Note: Custom Authorization Servers are available as part of the [API Access Management](/use_cases/api_access_management/) feature.
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Remove.

File: _source/_authentication-guide/auth-overview/index.md [hunk 4]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+
+| Feature                                  | Okta Authorization Server | Custom Authorization Server |
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Remove - I don't think this table presents any valuable info that isn't already covered by the above text.

File: _source/_authentication-guide/auth-overview/index.md [hunk 5]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+| Requires API Access Management           |                           | &#10004;                    |
+| Org Administrator creates one or more    |                           | &#10004;                    |
+| Validate the Access Token in custom code |                           | &#10004;                    |
+| Custom Scopes                            |                           | &#10004;                    |
+| Custom Claims                            |                           | &#10004;                    |
+| Custom Access Policies and Rules         |                           | &#10004;                    |
+
+For more information, see [Customizing Your Authorization Server](/authentication-guide/implementing-authentication/set-up-authz-server).
+
+If you use a Custom Authorization Server, we recommend that you use it instead of the Okta Authorization Server for any platform use cases (for example to perform SSO or secure your API). Doing so will make it easier to consume enhancements to the API Access Management product and features.
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Remove

File: _source/_authentication-guide/auth-overview/index.md [hunk 6]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+
+Okta provides a pre-configured Custom Authorization Server with the name `default`. This default authorization server includes a basic access policy and rule, which you can edit to control access.
+It allows you to specify `default` instead of the `authServerId` in requests to it:
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Suggestion for 312-316:

Okta provides a pre-configured Custom Authorization with the name `default`. It includes a basic access policy and rule to get you started quickly. To use the `default` Custom Authorization Server, use `default` as the authorization server ID:

`https://{yourOktaDomain}.com/api/v1/authorizationServers/${authServerId}` becomes
`https://{yourOktaDomain}.com/api/v1/authorizationServers/default`

For Custom Authorization Servers you create yourself, `${authServerId}` will be a random ID like `aus9o8wzkhckw9TLa0h7z`.

File: _source/_authentication-guide/auth-overview/index.md [hunk 7]
```diff
@@ -271,4 +271,56 @@ app -> client: Response
...
+
+* `https://{yourOktaDomain}.com/api/v1/authorizationServers/default` vs
+* `https://{yourOktaDomain}.com/api/v1/authorizationServers/${authServerId}` for other Customer Authorization Servers
+
+## OpenID Connect and Authorization Servers
```
Comment (nbarbettini) [inline _source/_authentication-guide/auth-overview/index.md]: Already covered in the "choosing your base URL" section, remove.

File: _source/_authentication-guide/auth-overview/index.md (hunk 6 above)
Comment (alexdahl-okta) [reply _source/_authentication-guide/auth-overview/index.md]: I know we haven't rolled out everywhere yet, but friendly reminder that we are moving to `https://{yourOktaDomain}`!

Comment (nbarbettini) [reply _source/_authentication-guide/auth-overview/index.md]: Thank you @alexdahl-okta. It is to my eternal shame that I wrote a URL without catching that. 😭

**Pull request merged**